integration-cli/docker_cli_commit_test.go

```diff
@@ -49,10 +50,9 @@ func (s *DockerSuite) TestCommitPausedContainer(c *check.C) {
 	out, _ = dockerCmd(c, "commit", cleanedContainerID)
 
 	out, err := inspectField(cleanedContainerID, "State.Paused")
-	c.Assert(err, check.IsNil)
-	if !strings.Contains(out, "true") {
-		c.Fatalf("commit should not unpause a paused container")
-	}
+	c.Assert(err, checker.IsNil)
```

Comment:
@aditirajagopal almost there only this comment to go!

Thanks for doing all this!